man/fi_av.3.md

```diff
@@ -387,11 +407,17 @@ restrict receive buffers to a specific endpoint address and authorization key.
   will be updated to 0.  Failures will contain a fabric errno code.
 
 - *FI_AV_USER_ID*
-: This flag associates a user-assigned identifier with each AV entry
-  that is returned with any completion entry in place of the AV's address.
+: For address vectors configured without the FI_AV_USER_ID specified,
+  this flag associates a user-assigned identifier with each AV entry that
+  is returned with any completion entry in place of the AV's address.
+  Requesting this flag during insert may result in unexpected runtime
+  failures should a provider not support FI_AV_USER_ID with insert.
```

Comment:
Need to update documentation status that FI_AV_USER_ID will not be supported as an insert flag if the AV is opened with FI_AV_USER_ID.